test/test/helpers/use-i18n/test/index.js

```diff
@@ -1,32 +0,0 @@
-/**
```

Comment:
Agreed, we should keep it. However I'm not sure what to write here since the behavior of the i18n library changed. It's now initialized by default with the default locale and won't ever throw an error (even if you haven't call `.configure()`).